### ISSUE TITLE
account_owner role is not handled properly

### ISSUE THREAD

**Issue by houserx-dan** [SquadcastHub/terraform-provider-squadcast#15]
When trying to apply the account_owner role the squadcast API returns a response that the role is invalid:

```╷
│ Error: PUT https://api.squadcast.com/v3/users/61fd72c1cd079c3b576c27bf returned an error:
│ [400] role invalid
│ 
│   with squadcast_user.oncall_users["user@example.com"],
│   on squadcast.tf line 14, in resource "squadcast_user" "oncall_users":
│   14: resource "squadcast_user" "oncall_users" {
│ 
╵```

We ended up having to convert our account owner to a user so that this would not fail.

**Comment by caiohasouza** [comment]
+1

**Comment by roshan8** [comment]
@houserx-dan Thanks for reporting the issue. let me take a look.

**Comment by roshan8** [comment]
This has been addressed with provider version v1.0.5.
The `account_owner` role is now supported for `squadcast_user` resource.
